docs/contributor/arch/02-unified-metering-concept.md

```diff
@@ -0,0 +1,428 @@
+      "measure": {
+         "id": "ram_gb",
+         "value": 96
+      }
+   },
+   {
+      "id": "e2214e4e-9629-42a0-8fbb-5d204c29c21d",
+      "timestamp": "2022-11-14T09:30:00.339Z",
+      "consumer": {
+         "region": "cf-eu10",
+          "btp": {
+            "environment": "KUBERNETES",
+            "globalAccount": "dacce822-b68b-4cb8-9600-2ad630a10a17",
+            "subAccount": "96ce4d05-c1a3-46ae-aa9d-e6d0b07e0e6d",
+            "instance": "d6004172-fbc2-4e3c-b696-306a0ac62448"
+          }
+      },
+      "product": {
+         "service": {
+            "id": "xfs-kyma",
+            "plan": "standard"
+         }
+      },
+      "measure": {
+         "id": "persistent_volume_claims_size_gb_total",
+         "value": 150
+      },
+      "customDimensions": {
+         "count": 3,
+         "size_gb_rounded": 192,
+      }
+   },
+   {
+      "id": "f2214e4e-9629-42a0-8fbb-5d204c29c21d",
+      "timestamp": "2022-11-14T09:30:00.339Z",
+      "consumer": {
+         "region": "cf-eu10",
+          "btp": {
+            "environment": "KUBERNETES",
+            "globalAccount": "dacce822-b68b-4cb8-9600-2ad630a10a17",
+            "subAccount": "96ce4d05-c1a3-46ae-aa9d-e6d0b07e0e6d",
+            "instance": "d6004172-fbc2-4e3c-b696-306a0ac62448"
+          }
+      },
+      "product": {
+         "service": {
+            "id": "xfs-kyma",
+            "plan": "standard"
+         }
+      },
+      "measure": {
+         "id": "volume_snapshot_contents_size_gb_total",
+         "value": 150
+      },
+      "customDimensions": {
+         "count": 3,
+         "size_gb_rounded": 192,
+      }
+   },
+   {
+      "id": "g2214e4e-9629-42a0-8fbb-5d204c29c21d",
+      "timestamp": "2022-11-14T09:30:00.339Z",
+      "consumer": {
+         "region": "cf-eu10",
+          "btp": {
+            "environment": "KUBERNETES",
+            "globalAccount": "dacce822-b68b-4cb8-9600-2ad630a10a17",
+            "subAccount": "96ce4d05-c1a3-46ae-aa9d-e6d0b07e0e6d",
+            "instance": "d6004172-fbc2-4e3c-b696-306a0ac62448"
+          }
+      },
+      "product": {
+         "service": {
+            "id": "xfs-kyma",
+            "plan": "standard"
+         }
+      },
+      "measure": {
+         "id": "redis_tier",
+         "value": "S1"
+      },
+      "customDimensions": {
+         "count": 3,
+         "size_gb_total": 150,
+      }
+   }
+]
+```
+
+#### Benefits of Having a Persistent Cache and a Persistent Sending Queue
+
+A persistent cache ensures that cached data is not lost when the KMC restarts.
+KMC may restart for various reasons, such as upgrades or crashes due to exceeding memory limits.
+
+Similarly, a persistent sending queue prevents the loss of `UM measurements` if KMC is temporarily unable to send them to UM.
+This could happen due to network issues or if the UM service is down. During this time, the `Runtime Reconciler` will continue scraping resources and adding `UM measurements` to the Sending Queue.
```

Comment:
```suggestion
This could happen due to network issues or if the UM service is down. During this time, the `Runtime Reconciler` continues scraping resources and adding `UM measurements` to the Sending Queue.
```